packages/um7/package.py

```diff
@@ -68,16 +61,11 @@ def _bld_path(self):
         """
         return "ummodel_hg3"
 
-
-    def _bld_cfg_path(self, opt_value):
+    def _bld_cfg_path(self):
```

Comment:
Can this be a member variable instead of a member function?

Reply:
Done.